plugins/modules/mso_schema_site_anp_epg_domain.py

```diff
@@ -222,14 +263,23 @@ def main():
         switching_mode=dict(type="str"),
         enhanced_lagpolicy_name=dict(type="str"),
         enhanced_lagpolicy_dn=dict(type="str"),
+        binding_type=dict(type="str", default="none", choices=["dynamic", "ephemeral", "none", "static"]),
```

Comment:
I am missing the following items:
- test updates in https://github.com/CiscoDevNet/ansible-mso/blob/master/tests/integration/targets/mso_schema_site_anp_epg_domain/tasks/main.yml
- example update with the new attributes

Could you please add this?

Reply:
Also if possible please add your information to the author section and the top of the file

Reply:
@fmabille09 was the testing part clear, do you need any assistance on this?